ui/Live.jsx

```diff
@@ -59,13 +59,14 @@ class Live extends React.Component {
                                     ? <span className="logged-in">Logged in as: {user.emails[0].address}</span>
                                     : <span className="logged-in">Not logged in</span>
                                 }
-
+                                {!Meteor.isProduction &&
                                 <span className="check-user">
                                     <input type="checkbox" onChange={this.onCheckForUser.bind(this)} />
                                     <span>
                                         Bypass exposure firewall
                                     </span>
                                 </span>
+                                }}
```

Comment:
wait, isn't this giving an error ?

Reply:
Oops, I didn't actually test it, I just quickly edited on GitHub as a suggestion :sweat_smile: